src/cosalib/build.py

```diff
@@ -252,7 +255,8 @@ def refresh_meta(self):
 
     @property
     def basearch(self):
-        return self.meta.get("coreos-assembler.basearch", BASEARCH)
+        """ get the targetted architecture """
```

Comment:
targeted

Edit: apparently targetted is also acceptable, so... nevermind. :)

Reply:
i'm cool with either :) - how about `s/targetted/target/`?